### ISSUE TITLE
bug: doesn't handle empty last section in document

### ISSUE THREAD

**Issue by transitive-bullshit** [vhf/remark-lint-no-empty-sections#3]
Take this example:

```markdown
# Title

## Contents

- [Foo](#foo)

## Foo

non-empty

## License
```

The `License` section is empty, but it is not reported by this plugin.

**Comment by vhf** [comment]
Thanks for the report, I'll publish this fix as `3.0.0` in a couple of minutes.

**Comment by vhf** [comment]
Published: 611ec9fc7df0f7d75d246c941629b578ec750a00 / https://www.npmjs.com/package/remark-lint-no-empty-sections

**Comment by transitive-bullshit** [comment]
Thanks for the quick turnaround dude!
